src/rewritable_units/tokens/doctype.rs

```diff
@@ -86,14 +89,28 @@ impl<'i> Doctype<'i> {
     pub fn force_quirks(&self) -> bool {
         self.force_quirks
     }
+
+    /// Removes the doctype.
+    #[inline]
+    pub fn remove(&mut self) {
+        self.removed = true
```

Comment:
No need to add a new field here.
```suggestion
        self.raw = Bytes::from_str("", Encoding::UTF8);
```

Reply:
I can do this, but see above for the need to preserve the state as `bool` to match the behavior of other tags.

Reply:
Gotcha, no need to change it.